nautobot/ipam/templates/ipam/ipaddress_assign.html

```diff
@@ -3,12 +3,20 @@
 {% load form_helpers %}
 {% load helpers %}
 
+{% block title %}Assign an IP address{% endblock %}
+
 {% block content %}
-    <form action="{% querystring request %}" method="post" class="form form-horizontal">
-        {% csrf_token %}
+    <form action="#" method="get" class="form form-horizontal">
         {% for field in form.hidden_fields %}
             {{ field }}
         {% endfor %}
+        {% for k, v_list in request.GET.lists %}
+            {% if k != 'q' %}
+                {% for v in v_list %}
+                    <input type="hidden" name="{{ k }}" value="{{ v }}" />
```

Comment:
Can we use more meaningful variable names than `k` and `v`? It's not clear to me what these actually are. I *think* `k` is going to be "pk" and `v_list` is going to be a list of PK values?